tool/site/client-react/src/index.tsx

```diff
@@ -28,6 +29,9 @@ async function startApplication(baseUrl: string): Promise<void> {
         <PageContainer>
           <BrowserRouter>
             <Switch>
+              <Route path={`${baseUrl}change-auth`}>
+                <ChangeAuthPage/>
+              </Route>
```

Comment:
The original flow redirected to a default oauth login and once you were logged in the `change-auth` concept made sense, think it's lost meaning now that the redirect to change-auth is the default flow on initial visit to the site, `login` probably makes more sense than `change-auth` in the new context of the standard login flow

Reply:
yea we thought people would want to test the difference in functionality between user and system level access. so they will likely want to change-auth while already logged in. happy to change to login just would imply you should log out before logging in again (not really thought about log-out at all)

Reply:
yeah might be easier to make decisions once this is in int. future consideration maybe 👍